packages/blocks/src/surface-block/surface-block.ts

```diff
@@ -38,29 +38,37 @@ export class SurfaceBlockComponent extends LitElement {
   @property()
   host!: BlockHost;
 
-  firstUpdated() {
-    this.model.propsUpdated.on(() => this.requestUpdate());
-    this.model.childrenUpdated.on(() => this.requestUpdate());
-
+  private _initContainer() {
     const { page } = this.model;
     const yContainer = page.ySurfaceContainer;
     const container = new SurfaceContainer(this._canvas, yContainer);
     this._container = container;
 
     const params = new URLSearchParams(location.search);
     if (params.get('surface') !== null) {
```

Comment:
Put this logic into `playground`🤔

Reply:
Good catch, I think this can be a pattern to have not-enabled-by-default features in the playground.